subsystem/default_ingress_resource_test.go

```diff
@@ -338,6 +338,50 @@ var _ = Describe("default ingress", func() {
 		Expect(terraform.Apply()).To(BeZero())
 	})
 
+	It("Create cluster with default ingress - cluster_routes_hostname and cluster_routes_tls_secret_ref set to empty value", func() {
+		// Prepare the server:
+		server.AppendHandlers(
+			CombineHandlers(
+				VerifyRequest(http.MethodGet, "/api/clusters_mgmt/v1/clusters/123"),
+				RespondWithJSON(http.StatusOK, clusterReady),
+			),
+			CombineHandlers(
+				VerifyRequest(http.MethodGet, "/api/clusters_mgmt/v1/clusters/123"),
+				RespondWithJSON(http.StatusOK, clusterReady),
+			),
+			CombineHandlers(
+				VerifyRequest(http.MethodGet, "/api/clusters_mgmt/v1/clusters/123/ingresses"),
+				RespondWithJSON(http.StatusOK, defaultDay1Template),
+			),
+
+			CombineHandlers(
+				VerifyRequest(http.MethodPatch, "/api/clusters_mgmt/v1/clusters/123/ingresses/d6z2"),
+				VerifyJQ(`.cluster_routes_tls_secret_ref`, ""),
+				VerifyJQ(`.cluster_routes_hostname`, ""),
+				RespondWithJSON(http.StatusOK, `
+						{
+							"kind": "Ingress",
+							"href": "/api/clusters_mgmt/v1/clusters/123/ingresses/d6z2",
+							"id": "d6z2",
+							"listening": "external",
+							"default": true,
+							"dns_name": "redhat.com",
+							"load_balancer_type": "classic",
+							"route_wildcard_policy": "WildcardsDisallowed",
+							"route_namespace_ownership_policy": "Strict"
+						}
+					`),
+			),
+		)
+		// Run the apply command:
+		terraform.Source(`resource "rhcs_default_ingress" "default_ingress" {
```

Comment:
can you add 3 configs? one with both empty strings and 2 more for each one empty while the other is not. all 3 should fail, right?

Reply:
If we are adding tests were they are set individually I really don't think that an extra for both means much as the configuration is individual anyways :thinking: